tests/js/client/shell/shell-collection-api.js

```diff
@@ -30,8 +30,6 @@ const {arango, db, ArangoError} = require("@arangodb");
 const isCluster = require('internal').isCluster();
 const isEnterprise = require("internal").isEnterprise();
 const _ = require("lodash");
-const isWindows = (require("internal").platform.substr(0, 3) === 'win');
-const isServer = typeof require("internal").arango === 'undefined';
```

Comment:
Could you please clarify why the `isServer` has been removed from this test suite? I don't remember exactly what is used to represent, my guess would be a single server instance.

Reply:
Removing the `isServer` here is a change that is unrelated to Windows/MSVC support, so I could have done it in a separate PR.
At some point some other PR moved all tests from previously `tests/js/server` and `tests/js/common` to `tests/js/client`, so that the tests would all be executed inside arangosh (as opposed to inside arangod). 
The file you are referring to is also located in `tests/js/client`, so that it will always be executed inside an arangosh and never inside an arangod. Thus the `isServer` variable will always have a value of `false` here and can as well be removed entirely.